src/target/ext/base/multithread.rs

```diff
@@ -290,3 +313,25 @@ pub trait MultiThreadRangeStepping: Target + MultiThreadResume {
 }
 
 define_ext!(MultiThreadRangeSteppingOps, MultiThreadRangeStepping);
+
+/// Target Extension - support for GDB's "Scheduler Locking" mode.
+/// See [`MultiThreadResume::support_scheduler_locking`].
+pub trait MultiThreadSchedulerLocking: Target + MultiThreadResume {
+    /// Configure the target to enable "Scheduler Locking" for the upcoming
+    /// resume.
+    ///
+    /// This method is called when the GDB client sends a `vCont` packet that
+    /// **does not** include a default "continue" action (e.g.,
+    /// `vCont;s:pPID.TID` without a trailing `;c`).
+    ///
+    /// When this method is called, the implementation **must ensure that any
+    /// threads not explicitly resumed** via previous
+    /// `set_resume_action_...` calls **remain stopped**.
```

Comment:
```suggestion
    /// When this method is called, the implementation must ensure that any
    /// threads not explicitly resumed via previous
    /// `set_resume_action_...` calls **remain stopped**.
```

no need to double emphasis here